rust/foxglove/src/channel.rs

```diff
@@ -27,10 +27,18 @@ impl std::fmt::Display for ChannelId {
     }
 }
 
+/// A Schema is a description of the data format of messages in a channel.
+/// It allows Foxglove to validate messages and provide richer visualizations.
```

Comment:
Add a newline, so that the front-page "Structs" documentation captures just the first sentence.

```suggestion
/// A Schema is a description of the data format of messages in a channel.
///
/// It allows Foxglove to validate messages and provide richer visualizations.
```